src/etl/impl/LedgerPublisher.h

```diff
@@ -214,14 +219,21 @@ class LedgerPublisher
     lastCloseAgeSeconds() const
     {
         std::shared_lock lck(closeTimeMtx_);
-        auto now = std::chrono::duration_cast<std::chrono::seconds>(std::chrono::system_clock::now().time_since_epoch())
-                       .count();
-        auto closeTime = lastCloseTime_.time_since_epoch().count();
-        if (now < (rippleEpochStart + closeTime))
+        auto const now =
+            std::chrono::duration_cast<std::chrono::seconds>(std::chrono::system_clock::now().time_since_epoch())
+                .count();
+        auto const closeTime = lastCloseTime_.time_since_epoch().count();
+
+        if (auto const age = now - (rippleEpochStart + closeTime); age < 0)
```

Comment:
Looks like you could use `std::max` here for a one liner. Up to you:
```cpp
return std::max(0, now - (rippleEpochStart + closeTime));
```

Reply:
I tried this way . but if std::max<uint> the result will be wrong, if std::max<int> , we will update the return type.